app/scripts/directives/add-events/add-event/forms/birth-notification.json

```diff
@@ -17,17 +17,11 @@
           "required": true
         },
         "FHIRMappings": [{
-          "resourceKey": "location",
-          "path": "name",
-          "params": null,
-          "valueType": "formValue",
-          "value": "birthPlace"
-        }, {
           "resourceKey": "main",
           "path": "location[0].location.reference",
           "params": null,
-          "valueType": "staticValue",
-          "value": "@location"
+          "valueType": "formValue",
```

Comment:
this might need to be updated to `optionValue` to get the key for the selected value, `formValue` will give you the display value of the option